### ISSUE TITLE
template week-day.html 

### ISSUE THREAD

**Issue by sidneyalan** [Serhioromano/bootstrap-calendar#787]
template week-day.html shows the meetings one below the other in different columns, can it be made to show one next to the other in their respective columns?

**Comment by Neozxz** [comment]
十分感谢，邮件已经收到！

**Comment by cablegunmaster** [comment]
Yeah it can you just need to rewrite the whole page, as this is not supported currently but its worth a rewrite.
Basically rewrite the CSS and the html and use a different approach because currently it would just 
show next to each other. There is no simple solution for this since this is a limitation how the library currently is setup.

**Comment by sidneyalan** [comment]
Thanks for your answer, I thought that rewriting the cal-cell and cal-offset classes could solve it.
